### PR TITLE
Updated herolte device page

### PR DESCRIPTION
**Pull request by abkro**
Edited herolte.rst page with links to updated WIP device and kernel repos and did a couple of other minor edits to the page.

### CODE REVIEW

File: supplementary/devices/herolte.rst [hunk 1]
```diff
@@ -95,7 +95,7 @@ If at some point the device does not boot at all from the power off state, no ma
 
 * Then switch completely off, also as described above. 
 
-* Now, you should once more be able to either boot normally or boot into recovery, whichever you wish.
```
Comment (doniks) [inline supplementary/devices/herolte.rst:98]: s?

File: supplementary/devices/h815.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
+H815 - LG G4
+===========================
```
Comment (doniks) [inline supplementary/devices/h815.rst:2]: underline is too long

**Review by doniks — CHANGES_REQUESTED** [review]
two small remarks below. but more importantly, git says it conflicts!

**Comment by abkro** [comment]
I have dealt with the conflicts now as far as I can see. I don't understand the last one. Tried multiple times....

**Review by doniks — APPROVED** [review]

**Comment by doniks** [comment]
hi @abkro . I've resolved the conflicts. Had to add a column in the table (works/doesn't work) - please check

**Comment by doniks** [comment]
no objections. I'll merge

**Pull request merged**